bucket/manifold-viewer-9-official.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "9.0.174",
+    "description": "Official build of Manifold Viewer Release 9",
+    "homepage": "http://manifold.net/",
+    "license": "Unknown",
+    "depends": "extras/vcredist2017" ,
```

Comment:
Never depends on vcredists

Reply:
ok. (but why? the ability to declare dependencies was appealing. I've noticed that vcredist2017 scoop install acts wierd. Is that the reason?)